app/.eslintrc.js

```diff
@@ -31,5 +31,71 @@ module.exports = {
     'import/no-named-as-default': 0,
     'import/prefer-default-export': 0,
     'no-param-reassign': ['error', { props: false }],
+    'import/order': [
+      'warn',
+      {
+        groups: [['builtin'], 'external', 'internal', 'parent', 'sibling'],
+        'newlines-between': 'always',
+        alphabetize: {
+          order: 'asc',
+          caseInsensitive: true,
+        },
+        pathGroups: [
+          {
+            pattern: 'react',
+            group: 'builtin',
+            position: 'before',
+          },
+          {
+            pattern: 'react** || @react**',
+            group: 'builtin',
+          },
+          {
+            pattern: 'node_modules/**',
+            group: 'builtin',
+          },
+          {
+            pattern: 'next/**',
+            group: 'builtin',
+            position: 'after',
+          },
+          {
+            pattern: 'hoc/**',
+            group: 'external',
+            position: 'before',
+          },
+          {
+            pattern: 'store/**',
+            group: 'external',
+          },
+          {
+            pattern: 'hooks/**',
+            group: 'unknown',
```

Comment:
why `hooks` folder belongs to `unkown` group?

Reply:
I don't like that internal imports come before hooks

![image](https://user-images.githubusercontent.com/8928965/127300949-d8db32a5-1c6f-47b4-834c-3f5ffe81fe5b.png)

Reply:
Attached new proposal.